src/org/parosproxy/paros/extension/manualrequest/ManualRequestEditorDialog.java

```diff
@@ -206,6 +208,20 @@ public void run() {
                 try {
                 	getMessageSender().handleSendMessage(aMessage);                    
                     postSend();
+                } catch (IOException sslEx) {
+                    StringBuilder strBuilder = new StringBuilder();
+
+                    strBuilder.append(Constant.messages.getString("network.ssl.error.connect"));
+                    strBuilder.append(((HttpMessage) aMessage).getRequestHeader().getURI().toString()).append('\n');
+                    strBuilder.append(Constant.messages.getString("network.ssl.error.exception") + sslEx.getMessage())

```

Comment:
- Ideally it should also `append` the message of the exception.
 - It could use `ExceptionUtils.getRootCauseMessage(Throwable)` to obtain the message of the exception.